website/docs/newoption.md

```diff
@@ -10,11 +10,12 @@ newoption { description }
 
 | Field       | Description                                                                        |
 |-------------|------------------------------------------------------------------------------------|
-| trigger     | What the user would type on the command line to select the option, e.g. "--name". |
+| trigger     | What the user would type on the command line to select the option, e.g. `--name`. |
 | description | A short description of the option, to be displayed in the help text. |
 | value       | Optional. If the option needs a value, provides a hint to the user what type of data is expected. |
 | allowed     | Optional. A list of key-value pairs listing the allowed values for the option. |
 | default     | Optional. Sets the default for this option if not specified on the commandline. |
+| category    | Optional. Places the option under a seperate header when the user passes `--help` |
```

Comment:
```suggestion
| category    | Optional. Places the option under a separate header when the user passes `--help` |
```

Reply:
Thanks! Fixed with a force push to minimise commits. Sorry if that means you miss out on credit...